src/igmpv3.h

```diff
@@ -21,6 +21,21 @@
 *   igmpv3.h - Header file for common IGMPv3 includes.
 */
 
+/*
+ * IGMP v3 query format.
+ */
+struct igmpv3 {
```

Comment:
I would suggest to rename this structure to `struct igmpv3_query` as _IGMPv3 Membership Query_ packets has different structure than _IGMPv3 Membership Report_ (which is already defined few lines below). Or do you have other opinion?

Reply:
I have named it according to xBSD "netinet/igmp.h", because they have a struct for IGMPv3 query already in that header that is named "igmpv3". To make the code compile on all platforms, I have copied the definition from xBSD to "os-linux.h", because linux is missing IGMPv3 structure definition in "netinet/igmp.h"

Reply:
There is already file `src/igmpv3.h` in igmpproxy project which contains other IGMPv3 structures. So IGMPv3 Membership Query should be there too and correctly named. Having incorrectly named structures just increase mess in the project. And if xBSD has this mess, it does not mean that it has to be also in igmpproxy.